api/secure/account.ts

```diff
@@ -124,16 +124,22 @@ export const updateAccountFollowingList =
 export const updateAccount = (urlArg: URL, getJwt: func_GetJwt) =>
 async (args: {
     npub: string;
-    account: {
-        about?: string;
-        banner?: string;
-        displayName?: string;
-        lud06?: string;
-        lud16?: string;
-        name?: string;
-        picture?: string;
-        phone?: string;
-        website?: string;
+    // according to https://github.com/satlantis-dev/api/blob/7091602016803474021e2869b818d94993a812e4/rest/account.go#L20
+    update: {
+        about: string;
+        banner: string;
+        businessCategory: string;
+        displayName: string;
+        isBusiness: boolean;
+        lud06: string;
+        lud16: string;
+        name: string;
+        nip05: string;
+        picture: string;
+        phone: string;
+        website: string;
```

Comment:
@satoshisound It will be nicer if they are all optional fields. The UI does not necessary always have the whole user profile object / account object available in the scope of editing the profile.

Our mobile code also has a different architecture from the Svelte code, which has a global `$account`.